<?xml version="1.0"?>
<launch>
    <param name="use_sim_time" value="false"/>

    <!--Nodes-->
	<node name="path_planner" pkg="fs_planning-2022" type="path_planner.py" output="screen"/>

    <!--Topics-->
    <!--<param name="planner/map_topic" value="/perception/smornn/detected_cones_lm"/>-->
    <param name="planner/map_topic" value="/SLAM/cones"/>
    <param name="planner/waypoints_topic" value="/planning/path"/> 

    <param name="planner/use_cleaner" value="True"/>
</launch>

